### PR TITLE
Di 19898 handle uninstall

### PR DESCRIPTION
**Pull request by roiaharon-pepperi**

### CODE REVIEW

File: client-side/src/app/mapping/form/mapping-form.component.ts [hunk 1]
```diff
@@ -18,17 +20,65 @@ import { IPepGenericFormValueChange, PepGenericFormService } from '@pepperi-addo
 export class MappingFormComponent implements OnInit {
 
     item: MappingFormItem;
+    atdFields;
+
+    //dataSource: IPepGenericFormDataSource
```
Comment (michaelatias) [inline client-side/src/app/mapping/form/mapping-form.component.ts:25]: delete comments?

File: client-side/src/app/mapping/form/mapping-form.component.ts [hunk 2]
```diff
@@ -51,42 +101,136 @@ export class MappingFormComponent implements OnInit {
...
+    }
+
+    documentKeyMappingChanged(value, field, index) {
+        this.item.DocumentKeyMapping[index] = {
+            Key: field,
+            Value: value
+        }
+    }
+
+    // async onValueChanged(event: IPepGenericFormValueChange) {
```
Comment (michaelatias) [inline client-side/src/app/mapping/form/mapping-form.component.ts:190]: delete comments

Comment (roiaharon-pepperi) [reply client-side/src/app/mapping/form/mapping-form.component.ts]: done

File: client-side/src/app/mapping/mapping-list.module.ts [hunk 3]
```diff
@@ -59,10 +58,27 @@ export const routes: Routes = [
         PepTextareaModule,
         PepDialogModule,
         PepGenericFormModule,
+        PepSeparatorModule,
+        // TranslateModule.forChild({
```
[the remark this answers is not in the document]
Comment (roiaharon-pepperi) [reply client-side/src/app/mapping/mapping-list.module.ts]: done

**Pull request merged**